sources/packages/web/src/views/aest/institution/Programs.vue

```diff
@@ -21,77 +20,46 @@
         </body-header>
       </template>
       <content-group>
-        <DataTable
-          :value="institutionProgramsSummary.results"
-          :lazy="true"
-          :paginator="true"
-          :rows="DEFAULT_PAGE_LIMIT"
-          :rowsPerPageOptions="PAGINATION_LIST"
-          :totalRecords="institutionProgramsSummary.count"
-          @page="pageSortEvent($event)"
-          @sort="pageSortEvent($event)"
-          :loading="loading"
+        <toggle-content
+          :toggled="!loading && !institutionProgramsSummary.count"
         >
-          <template #empty>
-            <p class="text-center font-weight-bold">No records found.</p>
-          </template>
-          <Column
-            :field="ProgramSummaryFields.SubmittedDate"
-            header="Date Submitted"
-            :sortable="true"
+          <v-data-table
```

Comment:
The pagination is not working for me. Am I missing something?
The below programs page from institutions can be used as a reference. Can you please take a look?
https://github.com/bcgov/SIMS/blob/89d4bde5c5499a01afc2707385e90064ccc8cf86/sources/packages/web/src/views/institution/locations/programs/LocationPrograms.vue#L40

One main difference is the use the `v-data-table` vs `v-data-table-server`.